### PR TITLE
Fix coverage issue

### PR DESCRIPTION
**Pull request by MoOx**
It seems that coverage is not accurate for master and next branch is still fucked up. Trying to debug this mess.

### CODE REVIEW

**Comment by thangngoc89** [comment]
I think codecov has nothing to do with this. It takes the coverage report from codecov and send it to the cloud. You need to look into nyc too.

**Comment by MoOx** [comment]
As usual, the issue is a PEKAC one. It's not nyc nor codecov.

**Comment by thangngoc89** [comment]
Ok. I'm a little confused now. This is result from html reporter

73.81% Statements 513/695
67.11% Branches 302/450
61.43% Functions 86/140
73.73% Lines 494/670

**Comment by MoOx** [comment]
The badge show "branch" percentage. But master coverage is not complete (some files are missing, not sure why).

**Comment by MoOx** [comment]
If I cannot have something that make sense without wasting my precious time, I will just use another service. It's so boring to waste time on shit like this.

File: package.json [hunk 1]
```diff
@@ -1,7 +1,8 @@
 {
   "name": "statinamic",
   "version": "0.9.1",
-  "description": "A static website generator to create dynamic website using React components.",
+  "version": "0.9.0",
```
Comment (thangngoc89) [inline package.json]: @MoOx some merge conflict ?

Comment (MoOx) [inline package.json]: oh shit.

**Pull request merged**